hack/windows/Install-OVS.ps1

```diff
@@ -199,6 +199,28 @@ function InstallOVS() {
     SetEnvVar "Path" $env:Path
 }
 
+function InstallDependency() {
+    # Check if SSL library has been installed
+    $pathEntries = $env:Path.Split(";") | ForEach-Object {
+        if ((Test-Path "$_/ssleay32.dll" -PathType Leaf) -and (Test-Path "$_/libeay32.dll" -PathType Leaf)) {
+            Log "Found existing SSL library."
+            return
+        }
+    }
+    $SSLZip = "openssl-1.0.2u-x64_86-win64.zip"
+    $SSLDownloadURL = "https://indy.fulgan.com/SSL/$SSLZip"
```

Comment:
it this source reliable? should we verify md5 to ensure integrity?

Reply:
This version (1.0.2u) is used to compile opensource OVS binaries (https://github.com/openvswitch/ovs/blob/master/appveyor.yml#L18), and this site is included in the suggested third-party Windows OpenSSL binaries in https://wiki.openssl.org/index.php/Binaries.

Reply:
I have added the md5sum check step. If you think using github link Wenying mentioned is better, I can update it. @tnqn

Reply:
As long as we verify md5sum, either link works for me.